poetry/console/commands/env_command.py

```diff
@@ -1,10 +1,8 @@
-from typing import TYPE_CHECKING
-
-from .command import Command
+from typing import Optional
 
+from poetry.utils.env import Env
 
-if TYPE_CHECKING:
-    from poetry.utils.env import Env
```

Comment:
this pattern is used in the case that an unconditional import would cause a circular import. This isn't required here.